provision.py

```diff
@@ -105,6 +107,11 @@
 
 def main():
     run(["sudo", "apt-get", "update"])
+    run(["sudo", "apt-get", "-y", "install", "software-properties-common"])
+    run(["sudo", "add-apt-repository", "-y", "universe"])
+    run(["sudo", "add-apt-repository", "-y", "ppa:groonga/ppa"])
+
+    run(["sudo", "apt-get", "update"])
```

Comment:
This only suffices to install pgroonga in a development environment; we'll also need some code for installing it in a production environment.  I think the right way to do this would be to add a new (optional) puppet module under puppet/zulip/manifests/ intended to be used alongside `postgres_appdb.pp` that adds this PPA and depends on the package.  Let me know if you need help understanding how to do this properly.